<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticoloDialog</class>
 <widget class="QDialog" name="ArticoloDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Articolo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>680</width>
            <height>315</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Marca:</string>
               </property>
               <property name="buddy">
                <cstring>cb_marca</cstring>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Unità di misura:</string>
               </property>
               <property name="buddy">
                <cstring>cb_unitamisura</cstring>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="le_scorta"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Scorta minima:</string>
               </property>
               <property name="buddy">
                <cstring>le_scorta</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Codice Articolo:</string>
               </property>
               <property name="buddy">
                <cstring>le_cod_articolo</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="pb_marca_dlg">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cb_marca"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_modello"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Modello:</string>
               </property>
               <property name="buddy">
                <cstring>le_modello</cstring>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QComboBox" name="cb_unitamisura"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Codice a barre:</string>
               </property>
               <property name="buddy">
                <cstring>le_cod_barre</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_descrizione"/>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="cb_catmerce"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="le_cod_articolo"/>
             </item>
             <item row="6" column="2">
              <widget class="QToolButton" name="pb_unita_dlg">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QToolButton" name="pb_catmerce_dlg">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Categoria:</string>
               </property>
               <property name="buddy">
                <cstring>cb_catmerce</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Descrizione :</string>
               </property>
               <property name="buddy">
                <cstring>le_descrizione</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="le_cod_barre"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>31</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Listino</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>310</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Fornitore:</string>
               </property>
               <property name="buddy">
                <cstring>cb_fornitore</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Codice Fornitore:</string>
               </property>
               <property name="buddy">
                <cstring>le_cod_fornitore</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Quantita:</string>
               </property>
               <property name="buddy">
                <cstring>le_quantita</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Prezzo acquisto:</string>
               </property>
               <property name="buddy">
                <cstring>le_prezzo_acquisto</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Sconto fornitore:</string>
               </property>
               <property name="buddy">
                <cstring>le_sconto</cstring>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Ricarico:</string>
               </property>
               <property name="buddy">
                <cstring>le_ricarico</cstring>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Prezzo vendità:</string>
               </property>
               <property name="buddy">
                <cstring>le_prezzo_vendita</cstring>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>IVA</string>
               </property>
               <property name="buddy">
                <cstring>cb_codiva</cstring>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Prezzo finito:</string>
               </property>
               <property name="buddy">
                <cstring>le_prezzo_finito</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QToolButton" name="pb_fornitore_dlg">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="cb_fornitore"/>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="le_cod_fornitore"/>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="le_quantita"/>
             </item>
             <item row="3" column="1" colspan="2">
              <widget class="QLineEdit" name="le_prezzo_acquisto">
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="2">
              <widget class="QLineEdit" name="le_sconto"/>
             </item>
             <item row="11" column="1" colspan="2">
              <widget class="QLineEdit" name="le_prezzo_finito">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="10" column="1" colspan="2">
              <widget class="QLineEdit" name="le_prezzo_vendita">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QComboBox" name="cb_codiva"/>
             </item>
             <item row="9" column="2">
              <widget class="QLineEdit" name="le_iva">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="3">
              <widget class="QToolButton" name="pb_iva_dlg">
               <property name="text">
                <string>...</string>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Imponibile:</string>
               </property>
               <property name="buddy">
                <cstring>le_imponibile</cstring>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="2">
              <widget class="QLineEdit" name="le_imponibile">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="1" colspan="2">
              <widget class="QLineEdit" name="le_ricarico"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>2</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Note</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>680</width>
            <height>315</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Sede magazzino:</string>
               </property>
               <property name="buddy">
                <cstring>cb_sede</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Data arrivo:</string>
               </property>
               <property name="buddy">
                <cstring>le_data</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cb_sede"/>
             </item>
             <item row="0" column="2">
              <widget class="QToolButton" name="pb_sede_dlg">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="le_data">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Numero fattura:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_fattura"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>49</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Note:</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTextEdit" name="te_note"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_close">
       <property name="text">
        <string>Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clear">
       <property name="text">
        <string>Cancella</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save">
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>le_descrizione</tabstop>
  <tabstop>cb_marca</tabstop>
  <tabstop>pb_marca_dlg</tabstop>
  <tabstop>le_modello</tabstop>
  <tabstop>le_cod_articolo</tabstop>
  <tabstop>le_cod_barre</tabstop>
  <tabstop>cb_catmerce</tabstop>
  <tabstop>pb_catmerce_dlg</tabstop>
  <tabstop>cb_unitamisura</tabstop>
  <tabstop>pb_unita_dlg</tabstop>
  <tabstop>le_scorta</tabstop>
  <tabstop>cb_fornitore</tabstop>
  <tabstop>pb_fornitore_dlg</tabstop>
  <tabstop>le_cod_fornitore</tabstop>
  <tabstop>le_quantita</tabstop>
  <tabstop>le_prezzo_acquisto</tabstop>
  <tabstop>le_sconto</tabstop>
  <tabstop>le_ricarico</tabstop>
  <tabstop>cb_codiva</tabstop>
  <tabstop>pb_iva_dlg</tabstop>
  <tabstop>le_prezzo_finito</tabstop>
  <tabstop>cb_sede</tabstop>
  <tabstop>pb_sede_dlg</tabstop>
  <tabstop>le_data</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>te_note</tabstop>
  <tabstop>pb_save</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>le_iva</tabstop>
  <tabstop>le_prezzo_vendita</tabstop>
  <tabstop>pb_close</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>le_imponibile</tabstop>
  <tabstop>scrollArea_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>te_note</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_sconto</sender>
   <signal>editingFinished()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>updateImponibile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_prezzo_acquisto</sender>
   <signal>editingFinished()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>updatePrezzoAcquisto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_ricarico</sender>
   <signal>editingFinished()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>updateImponibile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_codiva</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>updateIva()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_prezzo_finito</sender>
   <signal>editingFinished()</signal>
   <receiver>ArticoloDialog</receiver>
   <slot>updatePrezzoFinito()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePrezzoAcquisto()</slot>
  <slot>updateIva()</slot>
  <slot>updatePrezzoFinito()</slot>
  <slot>updateImponibile()</slot>
 </slots>
</ui>
